src/main/java/com/microsoft/aad/msal4j/SilentParameters.java

```diff
@@ -28,6 +28,11 @@
 
     private IAccount account;
 
+    /**
+     * Claims to be requested through the OIDC claims request parameter, allowing requests for standard and custom claims
+     */
+    private ClaimsRequest withClaims;
```

Comment:
for silent API this value will be ignored if we just return token form cache - might be miss-leading